From ba242009e67638d5139a54a7c49eb6eaa6926bd9 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Sun, 26 Jul 2015 19:10:39 +0200
Subject: lua-factory: Disable "Music" support until GOA has support

This fixes the build for the current master gnome-online-accounts.

See https://bugzilla.gnome.org/show_bug.cgi?id=751981

diff --git a/src/lua-factory/grl-lua-factory.c b/src/lua-factory/grl-lua-factory.c
index f3b1e16..675e85f 100644
--- a/src/lua-factory/grl-lua-factory.c
+++ b/src/lua-factory/grl-lua-factory.c
@@ -664,15 +664,6 @@ enable_goa_source (GrlLuaGoaData *lua_data,
     if (goa_account_get_photos_disabled (acc) &&
         g_hash_table_contains (lua_data->sources, *source_id_suffix))
       return GOA_REMOVE;
-  } else if (g_strcmp0 (feature, "music") == 0) {
-    if (!goa_object_peek_music (object))
-      return GOA_NOTHING;
-    if (!goa_account_get_music_disabled (acc) &&
-        !g_hash_table_contains (lua_data->sources, *source_id_suffix))
-      return GOA_ADD;
-    if (goa_account_get_music_disabled (acc) &&
-        g_hash_table_contains (lua_data->sources, *source_id_suffix))
-      return GOA_REMOVE;
   } else if (g_strcmp0 (feature, "read-later") == 0) {
     if (!goa_object_peek_read_later (object))
       return GOA_NOTHING;
@@ -891,7 +882,6 @@ validate_account_feature (const char *lua_account_feature)
   const char const *features[] = {
     "photos",
     "read-later",
-    "music",
     NULL
   };
 
-- 
cgit v0.10.2

